snapshots/output/basic-project/lib/relationships.dart

```diff
@@ -29,6 +43,14 @@
 //      relationship scip-dart pub dart_test 1.0.0 lib/`relationships.dart`/SwimAction# implementation
 //                  ^^^^^^ reference scip-dart pub dart_test 1.0.0 lib/`relationships.dart`/Animal#
 //                              ^^^^^^^^^^ reference scip-dart pub dart_test 1.0.0 lib/`relationships.dart`/SwimAction#
+    @override
+//   ^^^^^^^^ reference scip-dart pub dart:core 2.19.0 dart:core/`annotations.dart`/override.
+    String field = 'otherVal';
+//  ^^^^^^ reference scip-dart pub dart:core 2.19.0 dart:core/`string.dart`/String#
+//         ^^^^^ definition scip-dart pub dart_test 1.0.0 lib/`relationships.dart`/Dog#field.
+//         relationship scip-dart pub dart_test 1.0.0 lib/`relationships.dart`/Animal#field. implementation reference
+//         relationship scip-dart pub dart_test 1.0.0 lib/`relationships.dart`/Mammal#field. implementation reference
```

Comment:
`field` is overriding the same field both `Animal` AND `Mammal`, relationship is correctly recorded